packages/crayons-core/src/components/data-table/data-table.tsx

```diff
@@ -905,6 +918,51 @@ export class DataTable {
     return closestCell;
   }
 
+  renderColResizer() {
+    return (
+      <div
+        role='none'
+        class='resizer'
+        onMouseDown={(e) => this.resizeCol(e)}
+      ></div>
+    );
+  }
+
+  setColResizerHeight() {
+    const resizers = this.el.shadowRoot.querySelectorAll('.resizer');
+    if (resizers.length > 0) {
+      resizers.forEach((item: HTMLElement) => {
+        item.style.height =
+          this.el.shadowRoot.querySelector('table').offsetHeight + 'px';
```

Comment:
can u move this line above the forEach statement. This is create reflow for every iteration since we are reading offsetHeight on each iteration. This value will remain constant.